### DISCUSSION TITLE
Installing Helix from homebrew says it install version 23.05, but it is apparently not

### DISCUSSION THREAD

**Discussion by aromeronavia** [helix-editor/helix#7559] [Troubleshooting] [answered]
## How did I discover this?
I wanted to enable the new multi language servers feature under languages.toml, and I find out I wasn't able to do so because the following error appeared.
```
Bad language config: unknown field `language-servers`, expected one of `name`, `scope`, `file-types`, `shebangs`, `roots`, `comment-token`, `text-width`, `soft-wrap`, `config`, `auto-format`, `formatter`, `diagnostic-severity`, `grammar`, `injectio
n-regex`, `language-server`, `indent`, `debugger`, `auto-pairs`, `rulers`, `workspace-lsp-roots`
in `language`
```

After digging a bit more, I looked into the [PR](https://github.com/helix-editor/helix/pull/2507) that added the change, and I noticed in the file changes that there were minor changes in the description of the following commands:
```
| `:lsp-restart` | Restarts the Language Server that is in use by the current doc |
| `:lsp-stop` | Stops the Language Server that is in use by the current doc |
-------------------------------- now became -------------------------------
| `:lsp-restart` | Restarts the language servers used by the current doc |
| `:lsp-stop` | Stops the language servers that are used by the current doc |
```

So, when I opened helix and looked for those copys, I found out there were different to the supposedly new ones
![image](https://github.com/helix-editor/helix/assets/13922607/128965d1-9251-485a-9229-214c6e604f16)

Notice the old copys remain in my version of Helix.

## My `hx --health` output.
```
Config file: /Users/albertoromero/.config/helix/config.toml
Language file: /Users/albertoromero/.config/helix/languages.toml
Log file: /Users/albertoromero/.cache/helix/helix.log
Runtime directories: /Users/albertoromero/.config/helix/runtime;/opt/homebrew/Cellar/helix/23.05/libexec/runtime;/opt/homebrew/Cellar/helix/23.05/libexec/bin/runtime
Runtime directory does not exist: /Users/albertoromero/.config/helix/runtime
Runtime directory does not exist: /opt/homebrew/Cellar/helix/23.05/libexec/bin/runtime
Clipboard provider: pbpaste+pbcopy
System clipboard provider: pbpaste+pbcopy
```

Any help is really appreciated 🙏 really want to start using the multiple LSP features.

> I did notice that 2 runtime directories that are "configured" under hx do not exist. One of them does, the `/opt/homebrew/Cellar/helix/23.05/libexec/runtime`

**Comment by the-mikedavis** [comment, marked as answer]
Support for multiple language servers was merged after the 23.05 release. It looks to me like you are running 23.05

**Reply by aromeronavia** [reply]
Just noticed that 🤦 I misread it. Thank you for the answer tho, I was about to facepalm this post too cuz just realized how dumb I am. Will just install it from source. Thank you!!
